.github/workflows/api-release.yml

```
@@ -39,0 +1,81 @@
+    if: ${{ !failure() && !cancelled() }}
+    needs: "build-image"
+    runs-on: ubuntu-latest
+    steps:
+      - name: Create GitHub Release
+        id: create_release
+        uses: softprops/action-gh-release@da05d552573ad5aba039eaac05058a918a7bf631 # v2.2.2
+        with:
+          tag_name: "${{ inputs.module }}_${{ github.event.inputs.release-version }}"
+          draft: false
+          prerelease: false
+          generate_release_notes: true
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)